### PR TITLE
production fixes

### PR DESCRIPTION
**Pull request by blunckr**
* move asset gems into dev folder, since they require extra things to be installed on the server.
* make sure dir where webpack build dumps stuff is created, but ignore the contents.

### CODE REVIEW

**Comment by bfcoder** [comment]
If this is change is necessary, should it go into https://github.com/atomicjolt/react_rails_starter_app instead? @jbasdf ?

**Comment by blunckr** [comment]
that's a good question. seesh, how often does travis fail on unrelated things? I think it's failing on 2 out of 2 PRs for me

File: .rubocop.yml [hunk 1]
```diff
@@ -387,9 +387,7 @@ Style/StringLiterals:
 Style/TrailingCommaInArguments:
   Description: 'Checks for trailing comma in argument lists.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas'
-  EnforcedStyleForMultiline:
```
Comment (blunckr) [inline .rubocop.yml]: setting these to an array is invalid, at least in the latest rubocop.

Comment (jbasdf) [inline .rubocop.yml]: I think we deal with most of the rubocop problems in the latest starter app code. We'll pull that in and it should take care of most of these errors.

**Pull request merged**